ui/js/graphs.js

```diff
@@ -720,16 +721,44 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
   $scope.loadingTestData = false;
 
   var testInputCreated = false;
-
+  var testArray = [];
+  var series = [];
   $scope.addTestData = function () {
-    var series = _.clone($scope.selectedSeries);
-    series.projectName = $scope.selectedProject.name;
-    $modalInstance.close(series);
+      var selectedSeries = $scope.rightList
```

Comment:
Assigning to a temporary variable here should not be necessary. Let's rename `$scope.rightList` to `$scope.selectedSeriesList`. That should make it more clear what needs to happen here.